### PR TITLE
Neural Coder enable VS Code extension

### PR DESCRIPTION
**Pull request by WenjiaoYue**
Signed-off-by: Yue, Wenjiao <wenjiao.yue@intel.com>

## Type of Change

add vscode_extension feature

## Description

add vscode_extension feature

## Expected Behavior & Potential Risk
can use in vscode

## How has this PR been tested?

search vscode in vscode market

## Dependency Change?
yes

Details are as follows：
@types/fs-extra，@types/mocha，@types/glob， @types/vscode，@typescript-eslint/eslint-plugin，@typescript-eslint/parser，@vscode/test-electroneslint，ts-loader，webpack，webpack-cli，python-shell

### CODE REVIEW

**Comment by kaiyaointel** [comment]
LGTM. When CI is passed, we can merge this.

**Review by kaiyaointel — APPROVED** [review]

**Comment by chensuyue** [comment]
You have added a list of 3rd party deps into BoM list, so pls fix this answer 'Dependency Change?'.

**Comment by WenjiaoYue** [comment]
> You have added a list of 3rd party deps into BoM list, so pls fix this answer 'Dependency Change?'.
yeah，I have fixed

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "lockfileVersion": 1
```
Comment (chensuyue) [inline package-lock.json:2]: Do we need this one?

**Review by chensuyue — APPROVED** [review]

**Comment by kaiyaointel** [comment]
@WenjiaoYue Could you take a look at the conflicts? I don't know what this is
image

**Comment by chensuyue** [comment]
Please rebase master to resolve this conflict.

**Pull request merged**